.changeset/two-turtles-march.md

```diff
@@ -0,0 +1,5 @@
+---
+'@web3-ui/components': minor
+---
+
+Added a TokenGate component that restricts access to child components unless erc-20/erc-721 token quantity requirements are met.
```

Comment:
Or not...we can't merge this one 😅  Create a new branch afresh pls!

Reply:
Haha alright for sure will do